spaces/S000038/properties/P000133.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000038
+property: P000133
+value: true
+refs:
+- mr: '3728284'
+  name: Topology (Munkres)
```

Comment:
It's not clear to use Munkres here.  Can you replace it with S&S, which is the place where we take the definition from?  And same for all the other files that use "By definition" if they come from S&S.